packages/h5p-server/src/types.ts

```diff
@@ -1533,6 +1576,13 @@ export interface IH5PConfig {
          */
         protocol?: 'http' | 'https';
     };
+    /*
+     * How often the user state of content is saved (in seconds). Set to false
+     * to disable saving user state. Note that the user state is only saved if
+     * the user object is passed into the render method of the player. You also
+     * must set ajax.contentUserData for state saving to work.
+     */
+    saveFreq: number | boolean;
```

Comment:
I think the config option should be named in a a clearer way. The name Joubel chose is unclear and doesn't really help.

Reply:
I agree. Any suggestions? I would suggest `contentUserDataSaveInterval` or `contentUserStateSaveInterval`.

Reply:
I'd opt for contentUserStateSaveInterval.